sql_metadb/derived_tables/instance_alternative_titles.sql

```diff
@@ -0,0 +1,31 @@
+--This derived table extracts data for the instance alternative titles. 
+--It includes the instance uuid, hrid, the alternative title, type, uuid, and the name associated with that type uuid.
+--Ordinality has been included. 
+DROP TABLE IF EXISTS instance_alternative_titles;
+
+CREATE TABLE instance_alternative_titles AS 
+SELECT 
+	i.id AS instance_id,
+	jsonb_extract_path_text(i.jsonb, 'hrid') AS instance_hrid,
+	jsonb_extract_path_text(titles.jsonb, 'alternativeTitle') AS alternative_title,
+    jsonb_extract_path_text(titles.jsonb, 'alternativeTitleTypeId')::uuid AS alternative_title_type_id,
+    titles.ordinality AS alternative_title_ordinality,
+    jsonb_extract_path_text(att.jsonb, 'name') AS alternative_type_name 

```

Comment:
```suggestion
    jsonb_extract_path_text(att.jsonb, 'name') AS alternative_title_type_name
```
To match with name in INDEX and to be consistent with naming.

Reply:
Thanks Eric. I just made that change.